[2016-05-24 00:57:03] local.ERROR: exception 'ErrorException' with message 'Undefined index: name' in E:\WWW\ninegroups\laravel\app\Http\Controllers\SearchController.php:9
Stack trace:
#0 E:\WWW\ninegroups\laravel\app\Http\Controllers\SearchController.php(9): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Undefined index...', 'E:\\WWW\\ninegrou...', 9, Array)
#1 [internal function]: App\Http\Controllers\SearchController->HomepageSearch()
#2 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Controller.php(246): call_user_func_array(Array, Array)
#3 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(162): Illuminate\Routing\Controller->callAction('HomepageSearch', Array)
#4 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(107): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\SearchController), Object(Illuminate\Routing\Route), 'HomepageSearch')
#5 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#6 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#7 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#8 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#9 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(108): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#10 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(67): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\SearchController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'HomepageSearch')
#11 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(198): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'HomepageSearch')
#12 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(131): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#13 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(691): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#14 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#15 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#16 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#17 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#18 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(693): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#19 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(660): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#20 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(618): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#21 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(210): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#22 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#23 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(43): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 E:\WWW\ninegroups\laravel\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#26 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#27 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(55): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#29 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(61): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#31 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(36): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#33 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(40): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#34 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#35 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(42): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#36 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#37 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#38 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#39 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(111): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#40 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(84): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#41 E:\WWW\ninegroups\laravel\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#42 {main}  
[2016-05-24 02:21:52] local.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] [2002] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
' in E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php:47
Stack trace:
#0 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php(47): PDO->__construct('mysql:host=192....', 'root', 'root', Array)
#1 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Database\Connectors\MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=192....', Array, Array)
#2 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Database\Connectors\ConnectionFactory.php(58): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Database\Connectors\ConnectionFactory.php(47): Illuminate\Database\Connectors\ConnectionFactory->createSingleConnection(Array)
#4 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Database\DatabaseManager.php(177): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#5 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Database\DatabaseManager.php(65): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#6 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Database\DatabaseManager.php(304): Illuminate\Database\DatabaseManager->connection()
#7 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(210): Illuminate\Database\DatabaseManager->__call('table', Array)
#8 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(210): Illuminate\Database\DatabaseManager->table('house')
#9 E:\WWW\ninegroups\laravel\app\Http\Controllers\SearchController.php(13): Illuminate\Support\Facades\Facade::__callStatic('table', Array)
#10 E:\WWW\ninegroups\laravel\app\Http\Controllers\SearchController.php(13): Illuminate\Support\Facades\DB::table('house')
#11 [internal function]: App\Http\Controllers\SearchController->HomepageSearch()
#12 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Controller.php(246): call_user_func_array(Array, Array)
#13 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(162): Illuminate\Routing\Controller->callAction('HomepageSearch', Array)
#14 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(107): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\SearchController), Object(Illuminate\Routing\Route), 'HomepageSearch')
#15 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#16 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(108): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#20 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(67): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\SearchController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'HomepageSearch')
#21 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(198): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'HomepageSearch')
#22 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(131): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#23 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(691): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(693): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#29 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(660): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#30 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(618): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#31 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(210): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#32 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#33 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(43): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#35 E:\WWW\ninegroups\laravel\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#36 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#37 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(55): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#38 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#39 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(61): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#40 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#41 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(36): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#42 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#43 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(40): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(42): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#46 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#47 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#48 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#49 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(111): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#50 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(84): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#51 E:\WWW\ninegroups\laravel\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#52 {main}  
[2016-05-24 02:22:32] local.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] [2002] 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。
' in E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php:47
Stack trace:
#0 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php(47): PDO->__construct('mysql:host=192....', 'root', 'root', Array)
#1 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Database\Connectors\MySqlConnector.php(20): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=192....', Array, Array)
#2 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Database\Connectors\ConnectionFactory.php(58): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Database\Connectors\ConnectionFactory.php(47): Illuminate\Database\Connectors\ConnectionFactory->createSingleConnection(Array)
#4 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Database\DatabaseManager.php(177): Illuminate\Database\Connectors\ConnectionFactory->make(Array, 'mysql')
#5 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Database\DatabaseManager.php(65): Illuminate\Database\DatabaseManager->makeConnection('mysql')
#6 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Database\DatabaseManager.php(304): Illuminate\Database\DatabaseManager->connection()
#7 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(210): Illuminate\Database\DatabaseManager->__call('table', Array)
#8 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(210): Illuminate\Database\DatabaseManager->table('house')
#9 E:\WWW\ninegroups\laravel\app\Http\Controllers\SearchController.php(13): Illuminate\Support\Facades\Facade::__callStatic('table', Array)
#10 E:\WWW\ninegroups\laravel\app\Http\Controllers\SearchController.php(13): Illuminate\Support\Facades\DB::table('house')
#11 [internal function]: App\Http\Controllers\SearchController->HomepageSearch()
#12 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Controller.php(246): call_user_func_array(Array, Array)
#13 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(162): Illuminate\Routing\Controller->callAction('HomepageSearch', Array)
#14 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(107): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\SearchController), Object(Illuminate\Routing\Route), 'HomepageSearch')
#15 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#16 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(108): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#20 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(67): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\SearchController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'HomepageSearch')
#21 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(198): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'HomepageSearch')
#22 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(131): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#23 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(691): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(693): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#29 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(660): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#30 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(618): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#31 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(210): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#32 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#33 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(141): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(43): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#35 E:\WWW\ninegroups\laravel\app\Http\Middleware\VerifyCsrfToken.php(17): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#36 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#37 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(55): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#38 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#39 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(61): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#40 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#41 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(36): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#42 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#43 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(40): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#44 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#45 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(42): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#46 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(125): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#47 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#48 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(101): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#49 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(111): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#50 E:\WWW\ninegroups\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(84): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#51 E:\WWW\ninegroups\laravel\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#52 {main}  
[2016-05-24 07:46:18] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '<<' (T_SL)' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:2
Stack trace:
#0 {main}  
[2016-05-24 07:46:18] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '<<' (T_SL)' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:2
Stack trace:
#0 {main}  
[2016-05-24 07:46:18] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '<<' (T_SL)' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:2
Stack trace:
#0 {main}  
[2016-05-24 07:46:18] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '<<' (T_SL)' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:2
Stack trace:
#0 {main}  
[2016-05-24 07:46:19] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '<<' (T_SL)' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:2
Stack trace:
#0 {main}  
[2016-05-24 07:46:19] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '<<' (T_SL)' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:2
Stack trace:
#0 {main}  
[2016-05-24 07:46:19] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '<<' (T_SL)' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:2
Stack trace:
#0 {main}  
[2016-05-24 07:46:19] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '<<' (T_SL)' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:2
Stack trace:
#0 {main}  
[2016-05-24 07:46:19] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '<<' (T_SL)' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:2
Stack trace:
#0 {main}  
[2016-05-24 07:46:50] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '<<' (T_SL)' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:2
Stack trace:
#0 {main}  
[2016-05-24 07:52:04] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '<<' (T_SL)' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:2
Stack trace:
#0 {main}  
[2016-05-24 07:52:56] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:52:57] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:52:57] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:52:58] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:52:58] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:52:58] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:52:58] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:52:59] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:52:59] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:52:59] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:52:59] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:52:59] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:52:59] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:52:59] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:52:59] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:52:59] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:53:00] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:53:00] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:53:00] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:53:00] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:53:00] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:53:00] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:53:00] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:53:00] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:53:00] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:53:00] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:53:00] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:53:00] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:53:00] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:53:00] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:53:00] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
[2016-05-24 07:53:00] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use DB as DB because the name is already in use' in E:\WWW\ninegroups\laravel\app\Http\Controllers\IndexController.php:6
Stack trace:
#0 {main}  
